trieste/models/gpflow/__init__.py

```diff
@@ -20,4 +20,5 @@
 from .config import GPflowModelConfig
 from .interface import GPflowPredictor
 from .models import GaussianProcessRegression, SparseVariational, VariationalGaussianProcess
+from .optimizer import _create_loss_function_external, _create_loss_function_internal
```

Comment:
can you remove this change please, this should be part of the optimizer refactoring PR